### PR TITLE
Borg changes. Compresses Service/Janitor/Scrubpuppy borgs together. Removes borgi. Adds vore sleepers to service and engi dogborgs. Demodularizes some dogborg stuff too.

### PR DESCRIPTION
**Pull request by BlackMajor**
Hey let me lay it down for ya.

-Removes Borgi as a module and moves the sprite to a Peacekeeper variant.
-Removes Janitor and Scrubpuppy borgs and moves all their sprites and equipment to Service borgs.
-Demodularizes some dogborg stuff.
-Fixes a couple of dogborg sprites.
-Adds a "dogborg_equip" proccall that equips flavour versions of dogborg noses and tongues. So you can still be cute, but in no way mechanically superior by being a dogborg.
-Adds a vore sleeper to engi and service dogborgs.

Did I miss anything?

TODO: Fix upgrades yeeting the dogborg flavour items
Apparantly fix something wrong with some sprites.

### CODE REVIEW

**Comment by r4d6** [comment]
what is the Engi vore Sleeper?

**Comment by Ghommie** [comment]
> -Adds a vore sleeper to engi and service dogborgs.

Please no, that thing is badly coded fetish content.

**Comment by Tupinambis** [comment]
Borgi is gone :crab:

**Comment by BlueWildrose** [comment]
**Alright, so I found a few bugs while this has been active:**
- The Janitor Marina appearance of the service module is missing her eyes at all times. They won't change whether she has her lights on or not; she always is just a blank screen. I don't know what else has been messed up in this regard besides Marina, but it's likely you should check the other borgs.
- When the engineering dog borg receives upgrades (like the RPED), they will lose their dog-based abilities. May want to also check what happens to the service dogborg if they receive upgrades like the advanced mop.

**And a request:**
The service borg now has the ability to clean and create messes as they have a spray can module. Their floor buffer is on at all times, which means it will actively remove any messes beneath them, such as graffiti that they made earlier. If possible, could the cyborg gain the ability to toggle this floor buffer on and off (probably through a module) for this?

**Comment by GrayRachnid** [comment]
ARE YOU TRYING TO SHADOW NERF PEACEKEEPER BORGS ON MY WATCH? NO YOU DON'T. GIVE ME BACK MY FIRE EXTINGUISHER.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -245,15 +275,15 @@
...
+		/obj/item/crowbar/cyborg,
 		/obj/item/reagent_containers/borghypo/epi,
 		/obj/item/healthanalyzer,
 		/obj/item/weldingtool/largetank/cyborg,
 		/obj/item/wrench/cyborg,
-		/obj/item/crowbar/cyborg,
 		/obj/item/stack/sheet/metal/cyborg,
 		/obj/item/stack/rods/cyborg,
 		/obj/item/stack/tile/plasteel/cyborg,
-		/obj/item/extinguisher,
```
Comment (GrayRachnid) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:256]: why are you

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -283,7 +315,6 @@
 		/obj/item/surgicaldrill,
 		/obj/item/scalpel,
 		/obj/item/circular_saw,
-		/obj/item/extinguisher/mini,
```
Comment (GrayRachnid) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: you

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 3]
```diff
@@ -372,12 +635,13 @@
...
 	basic_modules = list(
 		/obj/item/assembly/flash/cyborg,
+		/obj/item/extinguisher/mini,
+		/obj/item/crowbar/cyborg,
 		/obj/item/cookiesynth,
 		/obj/item/harmalarm,
 		/obj/item/reagent_containers/borghypo/peace,
 		/obj/item/holosign_creator/cyborg,
 		/obj/item/borg/cyborghug/peacekeeper,
-		/obj/item/extinguisher,
```
Comment (GrayRachnid) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:380]: doing this

**Comment by BlackMajor** [comment]
> ARE YOU TRYING TO SHADOW NERF PEACEKEEPER BORGS ON MY WATCH? NO YOU DON'T. GIVE ME BACK MY FIRE EXTINGUISHER.

Standardizing base borg equipment somewhat by giving all of them a crowbar/mini extinguisher/flash.

Logically, only engiborg should have the big fire extinguisher. What's peacekeeper need it for (aside from harm bashing, which the added crowbar does the same damage of)

**Comment by Ghommie** [comment]
>What's peacekeeper need it for (aside from harm bashing, which the added crowbar does the same damage of)

Because they already are a subpar medical/security/service hybrid module, there is nothing wrong with them having a fire extinguisher to help keep the station "peaceful".

**Comment by BlackMajor** [comment]
@BlueWildrose 
I'll add the buffer on/off switch in my next borg balance pass.

**Review by silicons — APPROVED** [review]

**Pull request merged**